.gitignore

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
This conflict needs to be solved.

Reply:
Resolve then push. There must have been a merge conflict.

Reply:
Resolved